CHANGELOG.md

```diff
@@ -137,6 +137,10 @@ v1.0.0-dev (unreleased)
     -   WithURLTemplate on HTTP outbounds **new**
     -   WithTracer on all transports **new**
 
+-   Introduced tracing and Context propagation support with OpenTracing for
+    transport authors.
+-   The `Serialize` package allows transport authors that require RPCs at rest
```

Comment:
Suggest "Created a `yarpc.serialize` package for marshalling RPC messages at rest. Useful for transports that persist RPC messages."